### PR TITLE
WFS Documentation

### PR DESCRIPTION
**Pull request by ranta**
Add field descriptions to WFS index page.
refs. LIIK-295

### CODE REVIEW

File: city_furniture/templates/wfs/feature_field.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<tr{% if level %} class="complex-field-member complex-level-{{ level }}"{% endif %}><th><code>{{ field.name }}</code></th><td>{{ field.xsd_element.type }}</td><td>{{ field.abstract|default:'' }}</td><td>{{ field.description|default:'-' }}</td></tr>
```
Comment (travenin) [inline city_furniture/templates/wfs/feature_field.html:1]: Quite a long line. Could add some line breaks to help readability.

File: city_furniture/templates/wfs/feature_type.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{% load i18n %}
+<h3>{{ feature_type.title }}</h3>
+{% if feature_type.abstract %}{{ feature_type.abstract|linebreaks }}{% endif %}
+
+{% block formats %}{% if wfs_output_formats %}
+  <p>
+    {% trans "Formats" %}:
+    {% for output_format in wfs_output_formats %}
+      <a href="?SERVICE=WFS&VERSION={{ version }}&REQUEST=GetFeature&TYPENAMES={{ feature_type.name }}&OUTPUTFORMAT={{ output_format.identifier }}">{{ output_format.title|default:output_format }}</a>{% if not forloop.last %},{% endif %}
```
Comment (travenin) [inline city_furniture/templates/wfs/feature_type.html:9]: Here too is unnecessarily long line.

File: city_furniture/templates/wfs/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<!doctype html>{% load i18n static %}{# This is a very basic template that can be overwritten in projects #}
```
Comment (travenin) [inline city_furniture/templates/wfs/index.html:1]: Is this your `{# comment #}`? Could be removed.

File: city_furniture/views/wfs.py [hunk 4]
```diff
@@ -58,35 +78,87 @@ class CityFurnitureWFSView(WFSView):
...
-        field("target_name_fi", model_attribute="target.name_fi"),
-        "content_responsible_entity",
-        "text_content_fi",
-        "text_content_sw",
-        "text_content_en",
-        "validity_period_end",
-        "validity_period_start",
-        "additional_material_url",
-        "lifecycle",
+        DescribedFeatureField("id", description=_("ID of the Furniture Signpost.")),
```
Comment (travenin) [inline city_furniture/views/wfs.py:81]: Are there translations somewhere for these strings? Or will we add them later?

**Review by travenin — CHANGES_REQUESTED** [review]
I want more readable HTML templates :)

File: city_furniture/views/wfs.py (hunk 4 above)
Comment (ranta) [reply city_furniture/views/wfs.py]: Yep, you're right, they are only intended to be in English. Removed `gettext_lazy` functions from description strings.

**Review by travenin — APPROVED** [review]

**Pull request merged**